app/code/Magento/Ui/Component/Form/Element/DataType/Media/Image.php

```diff
@@ -31,22 +31,30 @@ class Image extends Media
      */
     private $fileSize;
 
+    /**
+     * @var OpednDialogUrl
+     */
+    private $openDialogConfig;
```

Comment:
Let's keep consistent naming between class, property and variable to avoid confusion
```suggestion
    private $openDialogUrl;
```